.github/workflows/superset-e2e.yml

```diff
@@ -1,12 +1,12 @@
 name: E2E
 
 on:
-  push:
-    branches:
-      - "master"
-      - "[0-9].[0-9]*"
-  pull_request:
-    types: [synchronize, opened, reopened, ready_for_review]
+#  push:
```

Comment:
Okay I'm just going to assume you've got all of these commented out on purpose 😆

Reply:
These are workflows I would like to get working properly if we are going to maintain Superset long-term service hopefully just a temporary measure. Which reminds me I should create tickets to fix them....